hack/lib/build/version.sh

```diff
@@ -109,12 +109,9 @@ function os::build::version::kubernetes_vars() {
 	# Try to match the "git describe" output to a regex to try to extract
 	# the "major" and "minor" versions and whether this is the exact tagged
 	# version or whether the tree is between two tagged versions.
-	if [[ "${KUBE_GIT_VERSION}" =~ ^v([0-9]+)\.([0-9]+)(\.[0-9]+)*([-].*)?$ ]]; then
+	if [[ "${KUBE_GIT_VERSION}" =~ ^v([0-9]+)\.([0-9]+)\. ]]; then
```

Comment:
@stevekuznetsov @smarterclayton ptal

Reply:
Why?

Reply:
because it didn't actually work against the output of git describe, and all we care about is the major/minor bits

Reply:
Can you give the output of `git describe`? IIRC this will break stuff in OSE

Reply:
> Can you give the output of git describe?

`v1.10.0-46-g9070269`

> IIRC this will break stuff in OSE

nope, this is setting kube minor/major version, which are unset today. this isn't touching anything openshift-related